### DISCUSSION TITLE
Sending logs to remote syslog collector

### DISCUSSION THREAD

**Discussion by alwashali** [Security-Onion-Solutions/securityonion#1762] [Unsupported Versions] [answered]
Hello, 

I modified suricata logstash output pipeline under /opt/so/conf/logstash/pipelines/search to add syslog output

The problem is that after I restart the service using so-logstash-restart, my modifications get deleted and no changes saved 

I am just trying to send suricata logs to a syslog collector

**Comment by weslambert** [comment, marked as answer]
You need to modify in /opt/so/saltstack/local/salt/logstash/....... for it to be persistent.  Any custom modifications should go in `/opt/so/saltstack/local`.
